docs/api-reference/cli.md

```diff
@@ -48,6 +48,26 @@ NODE_OPTIONS='--inspect' next
 
 The first load is colored green, yellow, or red. Aim for green for performant applications.
 
+## Development
+
+`next dev` starts the application in development mode with hot-code reloading, error reporting, and more:
+
+The application will start at `http://localhost:3000` by default. The default port can be changed with `-p`, like so:
+
+```bash
+npx next dev -p 4000
+```
```

Comment:
Should we also say that you can add `-p 4000` to the script on `package.json`?

```js
"scripts": {
  "dev": "next -p 4000"
  ...
}
```

Reply:
Happy to add this in a follow up PR.